vita3k/config/src/config.cpp

```diff
@@ -276,4 +277,27 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
     return Success;
 }
 
+bool is_tracy_advanced_profiling_active_for_module(std::vector<std::string> &active_modules, std::string module, int *index) {
+    bool result = false;
+    // Retrieve index for module name in the list of enabled modules
+    auto iterator = std::find(active_modules.begin(), active_modules.end(), module);
+
+    // Check the index the iterator references to (`std::find()` returns `last` if no match was found)
+    if (iterator == active_modules.end()) {
+        // Return false if no match is found in the active modules vector
+        result = false;
+    } else {
+        // Return true if there was at least one match
+        result = true;
+    }
+
+    // If index is being requested and the module name was found
+    if ((index != nullptr) && (result)) {
+        // Calculate the distance between the first element in the module and
+        *index = std::distance(active_modules.begin(), iterator);
+    }
+
+    return result;
+}
+
```

Comment:
Should I warp this one in a `#ifdef TRACY_ENABLE ... #endif` check as well so it won't get compiled on Release builds or should I leave it as it is right now? The only code that executes it is in `vita3k/gui/src/settings_dialog.cpp` and that code is already set not to be compiled on Release builds.

Reply:
I think yes. ifdef is a visible sign, that this is needed for another build configuration and not a dead code